fabtests/common/shared.c

```diff
@@ -1598,6 +1602,9 @@ void ft_free_res(void)
 
 	ft_close_fids();
 
+	if (range_test_size) {
+		free (range_test_size);
+	}
```

Comment:
free() accepts a null pointer, no need to check.  Please check coding style.  The open paren should be next to free - no space.  Comment applies to other locations in the code.

Reply:
Done